apps/token-manager/app/src/App.js

```diff
@@ -86,100 +98,151 @@ class App extends React.PureComponent {
-                  onRemoveTokens={this.handleLaunchRemoveTokens}
-                />
-              ) : (
-                !isSyncing && (
-                  <EmptyState
-                    onActivate={this.handleLaunchAssignTokensNoHolder}
-                  />
-                )
-              )}
-            </AppLayout>
-            <SidePanel
-              title={
-                assignTokensConfig.mode === 'assign'
-                  ? 'Add tokens'
-                  : 'Remove tokens'
-              }
-              opened={sidepanelOpened}
-              onClose={this.handleSidepanelClose}
-              onTransitionEnd={this.handleSidepanelTransitionEnd}
+              <h1
+                css={`
+                  ${textStyle(layoutName === 'small' ? 'title3' : 'title2')};
+                  color: ${theme.content};
+                  margin-right: ${1 * GU}px;
+                `}
+              >
+                Tokens
+              </h1>
+              {tokenSymbol && <Badge.App>{tokenSymbol}</Badge.App>}
+            </div>
+          }
+          secondary={
+            <Button
+              mode="strong"
+              onClick={this.handleLaunchAssignTokensNoHolder}
+              css={`
+                ${layoutName === 'small'
+                  ? `
+                    width: ${5 * GU}px;
+                    height: ${5 * GU}px;
+                    min-width: 0;
+                    padding: 0;
+                  `
+                  : ''}
+              `}
             >
-              {appStateReady && (
-                <AssignVotePanelContent
-                  opened={sidepanelOpened}
-                  tokenDecimals={numData.tokenDecimals}
-                  tokenDecimalsBase={tokenDecimalsBase}
-                  onUpdateTokens={this.handleUpdateTokens}
-                  getHolderBalance={this.getHolderBalance}
-                  maxAccountTokens={maxAccountTokens}
-                  {...assignTokensConfig}
-                />
-              )}
-            </SidePanel>
-          </IdentityProvider>
-        </div>
-      </Main>
+              {layoutName === 'small' ? <IconPlus /> : 'Add tokens'}
+            </Button>
+          }
+        />
+
+        {appStateReady && holders.length > 0 ? (
+          <Holders
+            holders={holders}
+            groupMode={groupMode}
+            maxAccountTokens={maxAccountTokens}
+            tokenAddress={tokenAddress}
+            tokenDecimalsBase={tokenDecimalsBase}
+            tokenName={tokenName}
+            tokenSupply={tokenSupply}
+            tokenSymbol={tokenSymbol}
+            tokenTransfersEnabled={tokenTransfersEnabled}
+            userAccount={connectedAccount}
+            onAssignTokens={this.handleLaunchAssignTokens}
+            onRemoveTokens={this.handleLaunchRemoveTokens}
+          />
+        ) : (
+          !isSyncing && (
+            <EmptyState onActivate={this.handleLaunchAssignTokensNoHolder} />
```

Comment:
We should probably double check this and give it a light update @dizzypaty (super low priority; no normal user should ever see this empty state on the Tokens app)